common/src/main/java/com/wynntils/features/user/overlays/map/MinimapFeature.java

```diff
@@ -268,8 +268,8 @@ private void renderPois(
             compassOffsetX /= distance;
             compassOffsetZ /= distance;
 
-            float scaledWidth = width - 4 * compassSize;
-            float scaledHeight = height - 4 * compassSize;
+            float scaledWidth = width - 2 * compassSize;
```

Comment:
Are we maybe missing a parenthesis? Or am I tired..

Reply:
What parenthesis?

Reply:
Only the numbers have changed from 4 to 2

Reply:
(width -2) * compassSize?

Reply:
No, that is unintended as we are subtracting from width